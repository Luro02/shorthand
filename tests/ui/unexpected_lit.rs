use shorthand::ShortHand;

#[derive(ShortHand)]
#[shorthand(enable("string"))]
pub struct Command {
    value: String,
}

fn main() {}
