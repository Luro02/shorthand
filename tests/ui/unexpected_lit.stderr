error: unexpected literal type `string`
 --> $DIR/unexpected_lit.rs:4:20
  |
4 | #[shorthand(enable("string"))]
  |                    ^^^^^^^^
